README.md

```diff
@@ -56,6 +56,9 @@ Helm charts to install Istio and riff.
 ### Uninstall
 
 ```
+# remove any Knative deployers (if Knative runtime is enabled)
+kubectl delete --all-namespaces deployer.knative.projectriff.io --all
```

Comment:
Let's be more clear that all riff and knative (if installed) resource should be dropped

```suggestion
kubectl delete riff --all --all-namespaces
kubectl delete knative --all --all-namespaces
```